src/ucp/core/ucp_am.c

```diff
@@ -320,6 +317,17 @@ ucp_am_fill_short_header(ucp_am_hdr_t *hdr, uint16_t id, uint16_t flags,
     hdr->header_length = header_length;
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_am_fill_iov_elem(uct_iov_t *iov, void *buffer, size_t length,
+                     size_t *iov_cnt)
```

Comment:
should it be `iov_cnt_p` to emphasize this is a pointer to `size_t`?

Reply:
not sure, i'd say we use such syntax quite seldom

Reply:
ok